ext/standard/tests/strings/bug78840.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+Bug #78840 (imploding $GLOBALS crashes)
+--FILE--
+<?php
+$glue = '';
+@implode($glue, $GLOBALS);
```

Comment:
Are there any warnings generated by ``implode``?

Reply:
Yes, multiple "array to string conversion" notices.

Reply:
Ah yeah, forgot GLOBALS is an array of arrays, don't know what I was thinking.